<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryRecordDialog</class>
 <widget class="QWidget" name="HistoryRecordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>586</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>719</width>
    <height>586</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    border: 1px solid #000000;
   background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #FFFFFF, stop: 0.1 #A3A6A9,
                                 stop: 0.9 #A3A6A9, stop: 1.0 #FFFFFF);
}
TableView::item::selected
{
	color:white;
	background:blue;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="TitlebarWidget" name="titleBar" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>查找条件</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>联系人:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Edit_Relation"/>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>消息内容:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Edit_Msg"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushbtn_search">
              <property name="text">
               <string>查找</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushbtn_clear">
              <property name="text">
               <string>清除</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="childrenCollapsible">
            <bool>false</bool>
           </property>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="TableView" name="tableView" native="true">
               <property name="minimumSize">
                <size>
                 <width>707</width>
                 <height>150</height>
                </size>
               </property>
               <property name="showGrid" stdset="0">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="webview" name="webView" native="true">

           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="labelInfor">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="PbSearchAbove">
              <property name="minimumSize">
               <size>
                <width>45</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>上一条</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="PbSearchNext">
              <property name="minimumSize">
               <size>
                <width>45</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>下一条</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>346</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pbBegin">
              <property name="minimumSize">
               <size>
                <width>45</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>首页</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbblow">
              <property name="text">
               <string>上页</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbnext">
              <property name="text">
               <string>下页</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbEnd">
              <property name="text">
               <string>尾页</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>webview</class>
   <extends>QWidget</extends>
   <header location="global">webview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TitlebarWidget</class>
   <extends>QWidget</extends>
   <header>TitlebarWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QWidget</extends>
   <header>Tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
